.github/workflows/e2e.yml

```diff
@@ -10,35 +10,46 @@ on:
 jobs:
   e2e:
     timeout-minutes: 15
-    runs-on: ubuntu-latest
     strategy:
+      fail-fast: false
       matrix:
+        os: [ubuntu-latest, macos-latest]
         head: [headful, headless]
+    runs-on: ${{ matrix.os }}
     steps:
       - name: Checkout
         uses: actions/checkout@v3
-      - name: Install chrome-dev
-        run: |
-          wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | sudo apt-key add -
-          sudo sh -c 'echo "deb [arch=amd64] http://dl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/sources.list.d/google.list'
-          sudo apt-get update
-          sudo apt-get install -y google-chrome-unstable
       - name: Set up node.js
         uses: actions/setup-node@v3
         with:
           node-version: 16
           cache: npm
       - name: Install and build npm dependencies
         run: npm ci
+      - name: Install Chromium
+        id: chromium
+        run: |
+          version=$(npx @puppeteer/browsers install chrome@latest | cut -f 1 -d' ' | sed -e 's/chrome@//')
```

Comment:
Worth adding `@puppeteer/browsers` to devDependencies instead? Then we can just use `browsers` here, and it might be helpful when we testing/debugging locally anyhow.

Reply:
Done

Reply:
We don’t need `npx` anymore now that it’s a `devDependency`

Reply:
I forgot to update this comment, but I had done it and then it errored out, so in the end I undid the change.

You're saying that `browsers install chrome@latest` alone should work here, correct?
I can try again in a follow-up

Reply:
Hmm, I’m pretty sure this is what I’m doing in a bunch of GitHub Actions workflows I set up. But now that you mention it, it might be that I explicitly did `echo "$(npm bin)" >> $GITHUB_PATH` as part of the setup step (that also runs `npm ci`).